ui/lobby/src/view/realTime/filter.ts

```diff
@@ -66,9 +66,10 @@ export function toggle(ctrl: LobbyController, nbFiltered: number) {
     hook: bind('mousedown', ctrl.toggleFilter, ctrl.redraw),
     attrs: {
       'data-icon': ctrl.filterOpen ? 'L' : '%',
-      title: ctrl.trans.noarg('filterGames')
+      title: ctrl.trans.noarg('filterGames'),
+      'filtered': nbFiltered > 0 ? 'yes' : 'no'
```

Comment:
Toggle a class for that, instead of an arbitrary attribute.

Reply:
Fixed this in the newest commits.